fe/fe-core/src/main/java/com/starrocks/qe/MasterOpExecutor.java

```diff
@@ -71,6 +80,18 @@ public void execute() throws Exception {
             MysqlStateType state = MysqlStateType.fromString(result.state);
             if (state != null) {
                 ctx.getState().setStateType(state);
+                if (state == MysqlStateType.EOF || state == MysqlStateType.OK) {
+                    afterForward();
```

Comment:
If the follower execute failed, how to rollback it ?

Reply:
It is not easy to ensure that the global settings and session settings are consistent, because rollback may also fail. We can add some hints and how to eliminate these inconsistencies, if the follower execute failed.